config.go

```diff
@@ -23,7 +26,12 @@ func getJiraConfig() *JiraConfig {
 		return defaultConfig
 	}
 
-	file, err := os.Open("config.json")
+	// To get the path if user store the binary in $PATH
+	file, err := os.Open(getBinaryPath() + "/config.json")
```

Comment:
if user store the binary in $PATH, so, the config.json needs to follow ya? I don't think it is common for file such as config.json to be placed in /bin folder. In the case we want to separate the binary and the config, I think what we can do is to put the config.json into $HOME$/.jira. Wdyt?

Reply:
Oh yes agree with you kak. I guess, I can add more command to handle create config in that folder. Wdyt about this command?